build/static/boot/functions.sh

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env bash
+
 warning(){
 	echo "[WARNING] $1" 1>&2
 }
```

Comment:
Renamed the file because the filename using `aux` caused huge problems on Windows.